tools/driver/driver.cpp

```diff
@@ -71,7 +71,8 @@ extern int swift_format_main(ArrayRef<const char *> Args, const char *Argv0,
 /// \returns True if running as a subcommand.
 static bool shouldRunAsSubcommand(StringRef ExecName,
                                   SmallString<256> &SubcommandName,
-                                  SmallVectorImpl<const char *> &Args) {
+                                  const ArrayRef<const char *> &Args,
```

Comment:
Nitpick (here and below): ArrayRef is intended to be passed around by value; it's just a pointer/length pair.